CMakeLists.txt

```diff
@@ -59,7 +59,14 @@ add_compile_definitions(
 add_compile_options(/diagnostics:caret /W4 /WX /w14265 /w15038 /d1FastFail /guard:cf /Z7 /Gy /Zp8 /std:c++latest /permissive- /Zc:threadSafeInit- /Zl)
 
 set(VCLIBS_DEBUG_OPTIONS "/Od")
-set(VCLIBS_RELEASE_OPTIONS "/O2;/Os") # TRANSITION: Potentially remove /Os
+
+# TRANSITION: Potentially remove `/Os` if it results in speed improvements.
+# This requires benchmarking!
+# Note that in MSVC version 19.32.31328,
+# `/Os` results in a binary size difference of 102K
+# in the release DLL (a gain of 18.4%).
+# So, make certain that that tradeoff is considered when or if `/Os` is removed.
+set(VCLIBS_RELEASE_OPTIONS /O2 /Os)

```

Comment:
No change requested: I verified (with `ninja -v`) that changing `"/O2;/Os"` to `/O2 /Os` still results in `[...] /O2 /Os [...]` being passed to the compiler.